CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+* Support for `#[name = "foo"]` attribute in `#[pymethods]`. [#692](https://github.com/PyO3/pyo3/pull/692)
```

Comment:
Bikeshed: maybe the attribute should be `#[pyname]` instead of `#[name]`?

Reply:
`#[pyname]` could work, on the other hand, we already have `#[text_signature]` and not `#[pytext_signature]`, however `#[name]` is much more likely to conflict with something in the future due to being a much more common word than `#[text_signature]`